NitroxClient/Communication/Packets/Processors/DiscordRequestIPProcessor.cs

```diff
@@ -1,13 +1,33 @@
-﻿using NitroxClient.Communication.Packets.Processors.Abstract;
+﻿using System.Net;
+using System.Threading.Tasks;
+using NitroxClient.Communication.Packets.Processors.Abstract;
 using NitroxClient.MonoBehaviours.Discord;
+using NitroxModel.Helper;
 using NitroxModel.Packets;
 
 namespace NitroxClient.Communication.Packets.Processors;
 
 public class DiscordRequestIPProcessor : ClientPacketProcessor<DiscordRequestIP>
 {
-    public override void Process(DiscordRequestIP packet)
+    public override async void Process(DiscordRequestIP packet)
     {
-        DiscordClient.UpdateIpPort(packet.IpPort);
+        string ipPort = await GetIpPortAsync();
+        DiscordClient.UpdateIpPort(ipPort);
+    }
+
+    /** Get the WAN IP address or the Hamachi IP address if the WAN IP address is not available. */
+    private static async Task<string> GetIpPortAsync()
```

Comment:
If this is the same method as (used) in `LogHowToConnectAsync`, why not move it to NetHelper?

Reply:
I think this would get messy from an API design point. You'd need to return a list of IPs back but you don't know which is wan and which is hamachi unless you return a key-value pair back. Which imo is too much code for just 2 usages.